README.md

```diff
@@ -40,6 +40,7 @@ Then configure the rules you want to use under the rules section.
     "jest/no-disabled-tests": "warn",
     "jest/no-focused-tests": "error",
     "jest/no-identical-title": "error",
+    "jest/no-large-snapshots": "warn",
```

Comment:
no?

Reply:
I thought this was a list of all the rules exported that the user can configure independently of the `recommended` config. Why wouldn't it be in there?

Reply:
well, they now match the recommended config. Maybe that's weird? It doesn't list all rules

Reply:
Yeah you're right that is weird. As a user I took that as an example of how you could configure the rules exported within this plugin. To me the table near the bottom of the `README` is what lets me know whether a rule is in the `recommended` config or not.

Either way let me know how you want me to proceed.

Reply:
I don't really care, but having 2 lists of all rules doesn't make sense.

Take a look at e.g. https://github.com/yannickcr/eslint-plugin-react (just a couple examples, then a huge list). Or maybe https://github.com/benmosher/eslint-plugin-import, which has 5, then "etc.". followed by a list

Reply:
Makes sense. I was just trying to follow what I thought was the pattern already established. But I see that it wasn't a complete list to begin with so my bad! I will remove.